### PR TITLE
Add Sample component

### PR DESCRIPTION
**Pull request by penicillin0**
created CSS for sample.tsx

### CODE REVIEW

File: src/components/Sample.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import { FaRegCheckCircle, FaRegCircle } from "react-icons/fa";
+
+type Props = {};
+
+const Sample: React.FC<Props> = (props) => {
+  return (
+
+      <Container>
+        <Container_x>
+          <IconContext.Provider value={{ color: "mediumseagreen"}}><Container_a><FaRegCheckCircle/></Container_a></IconContext.Provider>
```
Comment (penicillin0) [inline src/components/Sample.tsx:14]: https://github.com/react-icons/react-icons#configuration

**Comment by penicillin0** [comment]
image

**Comment by penicillin0** [comment]
image

**Comment by penicillin0** [comment]
image

File: src/components/ProblemItemList.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  },
+  {
+    isSolved: false,
+    name: "三井住友信託銀行プログラミ...",
+    link: "https://atcoder.jp/contests/sumitrust2019/tasks/sumitb2019_d",
+  },
+  {
+    isSolved: true,
+    name: "三井住友信託銀行プログラミ...",
+    link: "https://atcoder.jp/contests/sumitrust2019/tasks/sumitb2019_d",
```
Comment (kdnk) [inline src/components/ProblemItemList.tsx:27]: url って名前にしたい気持ち。

**Pull request merged**